src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -11,9 +11,9 @@
 public class CommandResult {
 
     /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
+    private final String feedbackToUser;
 
-    /** The list of persons that was produced by the command */
+	/** The list of persons that was produced by the command */
```

Comment:
Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)